### PR TITLE
DBAAS-879: Code Maintenance

### PR DESCRIPTION
**Pull request by mrrajan**
Signed-off-by: Rajan Ravi <rravi@redhat.com>

Code maintenance for failed scripts and code optimization.

### CODE REVIEW

File: tests/cockroachdb_dev_provision.robot [hunk 1]
```diff
@@ -9,17 +9,18 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
 Test Setup          Given Setup The Test Case
 Test Teardown       Tear Down The Test Case
 Suite Teardown      Tear Down The Test Suite
+Force Tags          UI  cockroach
```
Comment (ikanias) [inline tests/cockroachdb_dev_provision.robot:12]: What does this tagging does? Does it mean that I can run this test using this tag?

Comment (mrrajan) [reply tests/cockroachdb_dev_provision.robot:12]: @ikanias Yes, we can target this test suite using -i or --include option on CLI.

File: tests/crunchydb_dev_provision.robot [hunk 2]
```diff
@@ -9,17 +9,18 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
 Test Setup          Given Setup The Test Case
 Test Teardown       Tear Down The Test Case
 Suite Teardown      Tear Down The Test Suite
+Force Tags          CLI     crunchy
```
Comment (ikanias) [inline tests/crunchydb_dev_provision.robot]: Why do you put CLI tagging here? Isn't it a UI test?
I think we should add the logout line in the teardown

Comment (mrrajan) [reply tests/crunchydb_dev_provision.robot]: Thanks for pointing that, It was by mistake and updated now.

**Review by ikanias — CHANGES_REQUESTED** [review]
Have some questions and a fix needed to be done.

**Review by ikanias — APPROVED** [review]
LGTM

**Comment by mrrajan** [comment]
Attached [logs](https://github.com/RHODA-lab/rhoda-ci/files/9925971/log.zip) only due to file size constraints

**Pull request merged**